about.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Green Earth Summit</title>
+</head>
+<body>
+    <nav>
+        <div class="upper-nav">
+            <ul class="top">
+                <li class="fb"><img  id="fb" src="assets/fb (1).png" alt="facebook icon"/></li>
+                <li class="twitter"><img id="twitter" src="assets/twitter.png" alt="twitter icon"></li>
+                <li class="pa"><p>English</p></li>
+                <li class="pa"><p>MyPage</p></li>
+                <li class="pa"><p>Logout</p></li>
+            </ul>
+        </div>
+        <div class="hamburger">
+            <ul>
+            <li class="bar"></li>
+            <li class="bar"></li>
+            <li class="bar"></li>
+        </ul>
+        </div>
+        <div  class="flex" >
+            <a  class="flex logo" href="index.html"><img  id="logo"  src="assets/Earth.png"/></a>
+            <ul class="flex nav-menu">
+              <li class="nav-item">
+              <a class="nav-link" id="nav1"  href="index.html">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Program</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">Join</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Sponsor</a>
+              </li>
+            <li class="nav-item">
+              <a class="nav-link"  href="#">News</a>
+            </li>         
+            <li class="nav-item GE">
+                <a class="nav-link" id="ge" href="#">Campaign</a>
+              </li>    
+          </ul>
+          </div>
+    </nav>
+   <!--Header-->
+   <header>
+    <h1 class="heading1about">"Hello!Sharing world"</h1>
+    <h2 class="heading2">Green Earth Global Summit 2023</h2>
+    <p  class="paragabout">Human existence is quite impossible without the presence of a healthy ecosystem. Our environment comprises all living and non-living components and their interactions within a natural habitat. Environmental conservation has become one of the core issues that need to be addressed to battle climate change and global warming.Green Earth Global aims at conservation of natural resources. </p>
+      </header>
```

Comment:
- [ ] Please not the design discrepancies in your implementation as highlighted below. I kindly suggest you fix them all. 😅 

**Design**
![top1](https://github.com/fatmahussein/Green-Earth-Summit/assets/99656687/be461ead-1124-4e3d-87c9-d6dbf029d9fe)

**Implementation**
![top2](https://github.com/fatmahussein/Green-Earth-Summit/assets/99656687/e1d0bd24-be16-443f-91b6-7c6fabd0d374)

- [ ] Note that the label **4** does not exist on the about page. You seem to have duplicated the home page here ⚠️ 
- [ ]  You have the contact information in a brow belt but that does not exist in the original design about page. kindly fix ⛑️ 

**PLEASE NOTE**
- [ ] The pages should look almost identical to the original design, outside the content differences.
- [ ] The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.

* *Basic color swatch: #272A31, #EC5242, #D3D3D3**

- [ ] The pages use the same fonts or simialr fonts from the original design.